charts/gateway-helm/templates/_helpers.tpl

```diff
@@ -39,7 +39,12 @@ helm.sh/chart: {{ include "eg.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+app.kubernetes.io/component: gateway-api
```

Comment:
i am not sure whether refer `gateway-api` as a component is accurate, any thoughts? 
cc @envoyproxy/gateway-maintainers

Reply:
IMO, default shouldn't

Reply:
eg is built on top of gateway-api and envoy.

Reply:
agree

Reply:
do you have an idea for the component label ?